### ISSUE TITLE
Please Add m3u8 support for Cache and Preloading like TIKTOK

### ISSUE THREAD

**Issue by Motasem-Hamed** [wxxsw/GSPlayer#58]

**Comment by phr85** [comment]
Yes I have same Issue. Did someone found workaround to cache m3u8 streams?

**Comment by jesuasir007** [comment]
https://github.com/garynewby/HLS-video-offline-caching/blob/main/HLSVideoCache/HLSVideoCache.swift

**Comment by jesuasir007** [comment]
can I use this cache mechanism

**Comment by jesuasir007** [comment]
hi any one know idea for preloading like ,mp4 file without cache that time load the m8u8 format but after the cache load set the mp4 file url set to avplayer ,can you use this idea and share a code please
